clients/mobile/src/utility/validator.ts

```diff
@@ -0,0 +1,67 @@
+interface PasswordPolicy {
+  minimum_length: number;
+  require_lowercase: boolean;
+  require_numbers: boolean;
+  require_uppercase: boolean;
+  require_symbols: boolean;
+}
+
+const passwordPolicy: PasswordPolicy = {
+  minimum_length: 10,
+  require_lowercase: true,
+  require_numbers: true,
+  require_uppercase: false,
+  require_symbols: true
+}
+
+export const isRequired = (value: string): string => {
+  return value.trim().length === 0 ? 'This field is required' : ''
+}
+
+export const isValidEmail = (value: string): string => {
+  const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+  return !emailRegex.test(value) ? 'Invalid email address' : ''
+}
+
+export const isValidPhoneNumber = (value: string): string => {
+  const phoneRegex = /^\+?[1-9]\d{1,14}$/
+  return !phoneRegex.test(value) ? 'Invalid phone number' : ''
+}
+
+export const isValidPassword = (value: string): string => {
```

Comment:
the name of the function suggests that if the password is valid, we would get a truthy value returned, if not we might get errors array. But in your case, I struggle to follow what it's doing.

Reply:
Update the name of the function.

You will find with this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/138/commits/94e59fc445b4f8c16e12ee97341d0810f25e56ab)